.env.example

```diff
@@ -1,2 +1,2 @@
 # Full base URL including protocol (http or https), host and optionally the port and the base api path without tailing forward slash
-BASE_API_ENDPOINT_URL=
\ No newline at end of file
+BASE_API_ENDPOINT_URL = 
```

Comment:
Are you sure this works with the parser as expected?

Also, I am not sure if I agree with the renaming.
It's not really an example if the value for the key is missing (that's why I went with template)

Reply:
`.env.example` is the standard. We can simply do it like this
`BASE_API_ENDPOINT_URL = http://localhost:3000/api` (depending on the API URL)

And yes it should work with the parser.